apps/wing-console/console/ui/src/layout/status-bar.tsx

```diff
@@ -65,11 +82,36 @@ export const StatusBar = ({
           </span>
         </div>
       </div>
+      {/*center*/}
+      <div className="w-full flex justify-center items-center"></div>
       {/*right side*/}
-      <div className="w-full flex space-x-1 justify-end">
+      <div className="w-full flex gap-2 items-center justify-end">
```

Comment:
```suggestion
      <div className="flex gap-2 items-center">
```

It isn't necessary for the div to grow, and `justify-end` doesn't have any effect here either.